assets/src/components/activities/adaptive/components/authoring/LayoutEditor.tsx

```diff
@@ -394,8 +394,17 @@ const LayoutEditor: React.FC<LayoutEditorProps> = (props) => {
     [dragSize, isDragging, selectedPartId],
   );
 
-  useKeyDown(() => setShowConfirmDelete(true), ['Delete', 'Backspace'], {}, []);
-  useKeyDown(handleCopyComponent, ['KeyC'], { ctrlKey: true }, [
+  useKeyDown(
+    () => {
+      if (selectedPartAndCapabilities && !configurePartId.length) {
```

Comment:
This is to make sure that the delete only gets triggered when a part is selected, and it is not in Edit mode.